packages/main/src/RadioButton.ts

```diff
@@ -0,0 +1,534 @@
+	@property()
+	name!: string;
+
+	/**
+	 * Defines the form value of the component.
+	 * When a form with a radio button group is submitted, the group's value
+	 * will be the value of the currently selected radio button.
+	 * <br>
+	 * <b>Important:</b> For the <code>value</code> property to have effect, you must add the following import to your project:
+	 * <code>import "@ui5/webcomponents/dist/features/InputElementsFormSupport.js";</code>
+	 *
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.main.RadioButton.prototype.value
+	 * @public
+	 */
+	@property()
+	value!: string;
+
+	/**
+	 * Defines whether the component text wraps when there is not enough space.
+	 * <br><br>
+	 * Available options are:
+	 * <ul>
+	 * <li><code>None</code> - The text will be truncated with an ellipsis.</li>
+	 * <li><code>Normal</code> - The text will wrap. The words will not be broken based on hyphenation.</li>
+	 * </ul>
+	 *
+	 * @type {sap.ui.webc.main.types.WrappingType}
+	 * @defaultvalue "None"
+	 * @name sap.ui.webc.main.RadioButton.prototype.wrappingType
+	 * @public
+	 */
+	@property({ type: WrappingType, defaultValue: WrappingType.None })
+	wrappingType!: WrappingType;
+
+	/**
+	 * Defines the accessible ARIA name of the component.
+	 *
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.main.RadioButton.prototype.accessibleName
+	 * @public
+	 * @since 1.6.0
+	 */
+	@property()
+	accessibleName!: string;
+
+	/**
+	 * Defines the IDs of the elements that label the component.
+	 *
+	 * @type {string}
+	 * @defaultvalue ""
+	 * @name sap.ui.webc.main.RadioButton.prototype.accessibleNameRef
+	 * @public
+	 * @since 1.1.0
+	 */
+	@property()
+	accessibleNameRef!: string;
+
+	@property({ defaultValue: "-1", noAttribute: true })
+	_tabIndex!: string;
+
+	/**
+	 * Defines the active state (pressed or not) of the component.
+	 * @defaultvalue false
+	 * @private
+	 */
+	@property({ type: Boolean })
+	active!: boolean;
+
+	/**
+	 * The slot is used to render native <code>input</code> HTML element within Light DOM to enable form submit,
+	 * when <code>name</code> property is set.
+	 * @type {HTMLElement[]}
+	 * @slot
+	 * @private
+	 */
+	@slot()
+	formSupport!: Array<HTMLElement>;
+
+	_deactivate: () => void;
+	_name!: string;
+	_checked!: boolean;
+
+	static i18nBundle: I18nBundle;
+
+	constructor() {
+		super();
+
+		this._deactivate = () => {
+			if (activeRadio) {
+				activeRadio.active = false;
+			}
+		};
+
+		if (!isGlobalHandlerAttached) {
+			document.addEventListener("mouseup", this._deactivate);
+			isGlobalHandlerAttached = true;
+		}
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get template() {
+		return RadioButtonTemplate;
+	}
+
+	static get styles() {
+		return radioButtonCss;
+	}
+
+	static get dependencies() {
+		return [Label];
+	}
+
+	static async onDefine() {
+		RadioButton.i18nBundle = await getI18nBundle("@ui5/webcomponents");
+	}
+
+	onBeforeRendering() {
+		this.syncGroup(false);
```

Comment:
better make the forceRemove an optional parameter and remove the argument false from here